scipy/stats/setup.py

```diff
@@ -1,5 +1,10 @@
 from os.path import join
 
+def pre_build_hook(build_ext, ext):
+    from scipy._build_utils.compiler_helper import get_cxx_std_flag
+    std_flag = get_cxx_std_flag(build_ext._cxx_compiler)
+    if std_flag is not None:
+        ext.extra_compile_args.append(std_flag)
```

Comment:
What is `ext`?

Reply:
`ext` is the Extension instance that is added eventually as a setup extension to compile

Reply:
Oops I see now that you you added `ext =` below. I missed that.